src/otx/algo/instance_segmentation/layers/transformer.py

```diff
@@ -91,11 +90,11 @@ class PatchEmbed(BaseModule):
 
     We use a conv layer to implement PatchEmbed.
 
+    TODO (sungchul): it is duplicated with otx.algo.modules.transformer.PatchEmbed
```

Comment:
If then, how about removing one and using other one in all code?

Reply:
How about doing this when refactoring instance segmentation?